### PR TITLE
Netlify deploy

### PR DESCRIPTION
**Pull request by jayvdb**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.2.3
```
Comment (Makman2) [inline .ruby-version]: why not 2.2.2 like described in the commit body?

File: Gemfile [hunk 2]
```diff
@@ -1,4 +1,5 @@
 source "https://rubygems.org"
 
 gem "jekyll"
+gem "jekyll-netlify"
```
Comment (Makman2) [inline Gemfile]: just curious, what kind of information does this plugin provide?

File: .ruby-version (hunk 1 above)
Comment (jayvdb) [reply .ruby-version]: It isnt supported by Netlify https://www.netlify.com/docs/continuous-deployment/#ruby

File: Gemfile (hunk 2 above)
Comment (jayvdb) [reply Gemfile]: https://rubygems.org/gems/jekyll-netlify
https://github.com/jayvdb/jekyll-netlify
(my first Ruby gem)

File: .ruby-version (hunk 1 above)
Comment (Makman2) [reply .ruby-version]: ah okay that was kind of irritating when reading the commit body^^

**Comment by yukiisbored** [comment]
ack https://github.com/coala/projects/commit/a36c5d0efab1df94d4bffe89fb2191de11170b1c

**Comment by yukiisbored** [comment]
ack https://github.com/coala/projects/commit/9367794e8e1bc1649f6266965a938faec1735aef

**Comment by yukiisbored** [comment]
ack https://github.com/coala/projects/commit/61ce023b9e3e1cdb16c5204632a1d7480e75b237

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**